docs/objective-c_abi.md

```diff
@@ -75,13 +75,13 @@ int result = objc_msgSend(receiver, selector, arg1, arg2);
 ```
 
 The call to `objc_msgSend` should not be performed as a variadic call but
-instead as if the `objc_msgSend` function had the same signature as the method
-that should be called, but with the two additional parameter, `self` and `op`,
+instead, as if the `objc_msgSend` function had the same signature as the method
```

Comment:
That comma seems wrong.

Reply:
https://www.englishgrammar.org/adverb-preposition/

Reply:
> When it goes at the beginning of a sentence, we usually separate it off with a comma.

It's not at the beginning of a sentence here, so it doesn't get a comma. The correct place to put the comma is after "but."